applications/directory/src/components/ThankYou.js

```diff
@@ -1,6 +1,6 @@
 import Card from '@mui/material/Card';
 
-function ThankYou() {
+const ThankYou = ({ name }) => {
```

Comment:
No need to make a separate reusable `ThankYou` component. Let's move this component inside the `ThankYouPage` and delete this file since it is small and will only be used there.